doc/plugin_agent_workloadattestor_systemd.md

```diff
@@ -0,0 +1,23 @@
+# Agent plugin: WorkloadAttestor "systemd"
+
+The `systemd` plugin generates selectors based on [systemd](https://systemd.io/) unit properties of the workloads calling the agent.
+
+This plugin does not accept any configuration options.
+
+General selectors:
+
+| Selector               | Value                                                                                                               |
+|------------------------|---------------------------------------------------------------------------------------------------------------------|
+| `systemd:Id`           | The unit Id of the workload (e.g. `systemd:Id:nginx.service`)                                                       |
+| `systemd:FragmentPath` | The unit file path this workload unit was read from (e.g. `systemd:FragmentPath:/lib/systemd/system/nginx.service`) |
```

Comment:
`systemd:fragment_path`

Reply:
I was going by the systemd naming convention of these [properties](https://www.freedesktop.org/software/systemd/man/org.freedesktop.systemd1.html). But I think it is better to switch to the format used by SPIRE plugins as you suggested.